lib/Db/Item.php

```diff
@@ -591,9 +591,11 @@ private function computeFingerprint(): string
      *
      * @return boolean
      */
-    public function isSupportedMime(string $mime): bool
+    public function isSupportedMime(?string $mime): bool
     {
+
         return (
+            $mime !== null ||
```

Comment:
This causes any valid string to return true. It should be
```php
$mime !== null && ( ... )
```

Reply:
And this is what I get for trying to rush some fixes. Thanks